README.md

```diff
@@ -86,6 +86,11 @@ require 'fakefs/safe'
 FakeFS.activate!(io_mocks: true)
 # your code
 FakeFS.deactivate!
+
+#or
+FakeFS(io_mocks: true) do
+  # your code
+end
```

Comment:
feels a bit overkill given that almost nobody will use this ...

Reply:
OK I removed my use of #478